sites/friday/src/get-started/quick-start/use-third-part.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Use a third party provider
+weight: 60
+description:   Welcome to our quick-start guide on hosting any stacks on {{< vendor/name >}} where we will demonstrate just how simple it is to host your projects on our PaaS. Follow the steps detailed below and you’ll have everything set up in no time.
+---
+
+## Use a third-party Git provider
+
+When you choose to use a third-party Git hosting service, the {{< vendor/name >}} Git
+repository becomes a read-only mirror of the third-party repository. All your
+changes take place in the third-party repository.
+
+Add an integration to your existing third-party repository:
+
+[//]: # (- [BitBucket]&#40;/integrations/source/bitbucket.md&#41;)
+- [GitHub](/integrations/source/github.md)
+
```

Comment:
we should add the [bitbucket](https://docs.upsun.com.fix-generic-gstarted-7jc6g4y-652soceglkw4u.eu-3.platformsh.site/integrations/source/bitbucket.html) and [gitlab](https://docs.upsun.com.fix-generic-gstarted-7jc6g4y-652soceglkw4u.eu-3.platformsh.site/integrations/source/gitlab.html) integrations as they are still valid with Upsun; they just can't be done via the console.